app/assets/javascripts/users.js

```diff
@@ -1,3 +1,11 @@
 // Place all the behaviors and hooks related to the matching controller here.
 // All this logic will automatically be available in application.js.
 // You can use CoffeeScript in this file: http://coffeescript.org/
+
+$(document).ready(function() {
+  var map;
+  map = new google.maps.Map(document.getElementById('map'), {
```

Comment:
Here you can assign the `map` variable in a single step:
`var map = new google.maps.Map(document.getElementById('map'), {`